application/models/team.py

```diff
@@ -0,0 +1,63 @@
+from application.errors import NotFound, NotInitialized
+from flask import current_app
+from py2neo import Graph, Node
+import uuid
+
+
+class Team:
+
+    @classmethod
+    def all(cls):
+        results = current_app.graph.run("MATCH (x:Team) RETURN x")
+        teams = []
+        for result in results:
+            team = Team(result['x']['name'])
+            team.id = result['x']['id']
+            teams.append(team)
+        return teams
+
+    @classmethod
+    def find_by(cls, force=False, id='', name=''):
+        team_node = current_app.graph.run(
+            f"MATCH (x:Team) WHERE x.name = '{name}' OR x.id = '{id}' RETURN x").evaluate()
```

Comment:
Is the code styling correct?
```
current_app.graph.run(
    ...).evaluate()
```
Looks pretty weird to me.

Reply:
Python doesn't do very well with method calls over multiple lines especially when the argument is only one string. I'll extract that String into a variable and pass that to the function instead, should make it more readible.

Reply:
If you put expressions in brackets, you can split them into multiple lines.
For strings you can use:
```
 ("first sentence"
  "second sentence")
```
which results in: "first sentencesecond sentence"

Reply:
This is outdated and splitting into multiple lines with braces is pretty much unreadable imo. I try to avoid it wherever possible.